src/bottomsheet/bottomsheet-common.ts

```diff
@@ -1,8 +1,8 @@
 import { View } from '@nativescript/core/ui/core/view';
-import { createViewFromEntry } from '@nativescript/core/ui/builder';
 import { Frame } from '@nativescript/core/ui/frame';
 import { EventData } from '@nativescript/core/data/observable';
 import { eachDescendant, ViewBase } from '@nativescript/core/ui/core/view-base';
+import { Builder } from "@nativescript/core";
```

Comment:
I don't want that change. No point in importing it all if we only use `createViewFromEntry`

Reply:
I chaged it, because the function itself is deprecated:
```@deprecated Use Builder.createViewFromEntry() instead.``` in ```@nativescript/core/ui/builder```

Reply:
@alexander-mai indeed! Did not see that. Good then